src/java.base/share/classes/java/util/Arrays.java

```diff
@@ -3789,6 +3815,14 @@ public static <T,U> T[] copyOfRange(U[] original, int from, int to, Class<? exte
         return copy;
     }
 
+
+    @ForceInline
+    private static void checkLength(int from, int to) {
+        if (to - from < 0) {
+            throw new IllegalArgumentException(from + " > " + to);
+        }
```

Comment:
Would it be beneficial to return the difference of `to` and `from` for caller use?

```suggestion
    private static int checkLength(int from, int to) {
        int newLength = to - from;
        if (newLength < 0) {
            throw new IllegalArgumentException(from + " > " + to);
        }
        return newLength;
```